README.md

```diff
@@ -1,2 +1,41 @@
-# my-Portfolio
- my portfolio website
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#   My Portfolio website
+
+- Project created to display my porfolio and to show my works
+
+
+## Built With
+
+- HTML & CSS
+- Linters
+
+👤 Collins Nana Antwi Baffour-Gyawu
```

Comment:
- [OPTIONAL] I think it's better to use `**Collins Nana Antwi Baffour-Gyawu**` as the author. Will create a more professional look for your README. For more information [click here](https://www.markdownguide.org/cheat-sheet/).